### ISSUE TITLE
corsika-reader - docs need some spicing up (Trac #1299)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1293]
everything renders in the same font makeing some things unclear (ie: `[]` looks like a "box placeholder" for unrenderable characters)

as small as it is, it might make more sense to do the docs solely in doxygen and have index.rst link to it.

release notes are a bit behind

Migrated from https://code.icecube.wisc.edu/ticket/1299
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:45", 
    "description": "everything renders in the same font makeing some things unclear (ie: `[]` looks like a \"box placeholder\" for unrenderable characters)\n\nas small as it is, it might make more sense to do the docs solely in doxygen and have index.rst link to it.\n\nrelease notes are a bit behind", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067105393059", 
    "component": "combo simulation", 
    "summary": "corsika-reader - docs need some spicing up", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2015-08-28T22:03:54", 
    "milestone": "", 
    "owner": "hpandya", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150828T22:04:09: 
* nega changed owner from "nega" to "olivas"
* nega changed component from "icerec" to "simulation"

**Comment by icecube-trac** [comment]
Trac update at 20151209T15:31:40: 
* olivas changed owner from "olivas" to "hpandya"
* olivas changed milestone from "" to "IceSim 5.0"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20160121T20:40:48: 
* david.schultz changed status from "assigned" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 141130:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141130"
> update release notes. clean up documentation formatting. closes #1293
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:45: 
* olivas changed milestone from "IceSim 5.0" to ""
* olivas commented:

> Milestone IceSim 5.0 deleted
